packages/backend/package.json

```
@@ -105,6 +104,7 @@
 		"body-parser": "1.20.3",
 		"bullmq": "5.26.1",
 		"cacheable-lookup": "7.0.0",
+		"canvas": "^3.1.0",
```

```suggestion
		"canvas": "3.1.0",
```

Misskeyのバージョン管理思想的にこっちの方が望ましいかも？

---

あーこれ適用したところでlockfileの再適用しないといけないんだった

---

done

---

> Misskeyのバージョン管理思想的にこっちの方が望ましいかも？

.npmrc に save-exact=true を指定するといいかも